test-library-models/src/main/java/com/uber/nullaway/testlibrarymodels/TestLibraryModels.java

```diff
@@ -66,7 +66,7 @@ public ImmutableSetMultimap<MethodRef, Integer> nullImpliesNullParameters() {
   @Override
   public ImmutableSet<MethodRef> nullableReturns() {
     return ImmutableSet.of(
-        methodRef("com.uber.Foo", "bar()"),
+        methodRef("com.uber.AnnotatedWithModels", "returnsNullFromModel()"),
```

Comment:
Renamed this to avoid clashing with other tests using the common name `Foo.bar()`